### PR TITLE
6 weekly forecast component

### LINKED ISSUE #6: Weekly forecast component

### PR DESCRIPTION
**Pull request by phfernandesmc**
in this branch I created the context to share the city weather information for the whole app and I also created the weekly weather forecast component

### CODE REVIEW

File: src/components/WeeklyComponent.tsx
Comment (Shingo023) [inline src/components/WeeklyComponent.tsx]: Since we are going to use SCSS Modules for the project, the class name should be like `{styles.WeeklyComponent}`?
If so, you also need to create and import `module.scss` file to style your component.

File: src/components/WeeklyComponent.tsx
Comment (Shingo023) [inline src/components/WeeklyComponent.tsx]: How about using 'weeklyWeather' in stead of 'days' as the variable name? I think that is more understandable.

File: src/components/WeeklyComponent.tsx
Comment (Shingo023) [inline src/components/WeeklyComponent.tsx]: How about 'WeeklyWeather' instead of 'days' and 'dailyWeather' instead of 'day'?

File: src/components/WeeklyComponent.tsx
Comment (Shingo023) [inline src/components/WeeklyComponent.tsx]: `className={styles.WeeklyComponentList}`

File: src/components/WeeklyComponent.tsx
Comment (Shingo023) [inline src/components/WeeklyComponent.tsx:57]: `className={styles.WeeklyComponentItem}`
`className={styles.WeeklyComponentDay}`

File: src/actions/weather.ts
Comment (Shingo023) [inline src/actions/weather.ts]: How about `displayedCityWeather` or something as the name of the let variable?
Since we also have to deal with the current location and its weather somewhere else, `currentCity` might sound a bit confusing.

File: src/actions/weather.ts
Comment (Shingo023) [inline src/actions/weather.ts]: How about `getDisplayedCityWeather`?

**Review by Shingo023 — APPROVED** [review]
Your code was very clean and readable!
I just left some comments related to naming of variables and class names.

If you are OK with them, I'd like you to modify them and merge your PR. I'm approving it now.

I appreciate your great work and patience!

**Review by Shingo023 — APPROVED** [review]
Thanks for the modifications!

File: src/components/WeeklyComponent.tsx
Comment (Saeka03) [inline src/components/WeeklyComponent.tsx]: I think the path here would be like this?
```suggestion
import styles from "../style/components/WeeklyComponent.module.scss";
```

File: src/components/WeeklyComponent.tsx
Comment (Saeka03) [inline src/components/WeeklyComponent.tsx]: Is this "dailyWeather" declared somewhere? Or you mean, like this?
```suggestion
          weeklyWeather.map((day, index) => (
```

File: src/components/WeeklyComponent.tsx
Comment (Saeka03) [inline src/components/WeeklyComponent.tsx]: Is this console.log for debugging?
If it's unnecessary, can you delete it?

File: src/components/WeeklyComponent.tsx
Comment (Saeka03) [inline src/components/WeeklyComponent.tsx]: According to Aoi's design, if it's today, the day should be "Today"?
But we can modify it after merging it!
image
image

File: src/style/components/WeeklyComponent.css
Comment (Saeka03) [inline src/style/components/WeeklyComponent.css]: Is this css file used somewhere?

File: src/style/components/WeeklyComponent.css.map
Comment (Saeka03) [inline src/style/components/WeeklyComponent.css.map]: Is this css file used somewhere?

File: src/style/components/WeeklyComponent.scss
Comment (Saeka03) [inline src/style/components/WeeklyComponent.scss]: I think the file name should be "WeeklyComponent.module.scss" to use scss in the WeeklyComponent.tsx file.

**Review by Saeka03 — APPROVED** [review]
Thank you for implementing! And I'm so sorry for being late to review your code.
I'll approve it and I'd appreciate it if you check my comments.

**Review by Saeka03 — APPROVED** [review]
LGTM! Thank you for modifying it 😄

**Review by Shingo023 — APPROVED** [review]

**Pull request merged**
